.github/workflows/bind-schemas-to-projects.yml

```diff
@@ -0,0 +1,166 @@
+name: Bind Schemas to HTAN2 Projects
+
+on:
+  repository_dispatch:
+    types: [schema-release]
+  workflow_dispatch:
```

Comment:
I wonfer if to start we ONLY want this to be on workflow-dispatch (ie manually) and move to repo dispatch once happy and stable